src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/CollectionCalendarService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities
+{
+    public class CollectionCalendarService : ICollectionCalendarService
+    {
+        private readonly HttpClient _client;
+
+        public CollectionCalendarService(HttpClient client)
+        {
+            _client = client;
+        }
+
+        public async Task ActivatePeriod(short calendarYear, byte periodNumber, bool active)
+        {
+            var url = "collectionCalendar/period/active";
```

Comment:
I think this url should just be collectionPeriods since an instance is what is being patched.